enforcer/params_utils.py

```
@@ -0,0 +1,35 @@
+    env_values = re.findall(r"{{[ ]*env\.(.*)[ ]*}}", value)
+    if env_values:
+        env_var_value = os.environ.get(env_values[0].strip(), None)
+        if not env_var_value:
+            msg = f"ENV var replacement {env_values[0]} does not exist for param: {value}"
+            logging.error(msg)
+            raise Exception(msg)
+        return env_var_value
+    return None
+
+
+def replace_env_vars_values(values: Dict) -> Dict:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)